### DISCUSSION TITLE
How To Make Widgets Dynamic In List Page

### DISCUSSION THREAD

**Discussion by synsyst** [filamentphp/filament#14330] [Help]
### Package

Widgets

### Package Version

v3.2.23

### How can we help you?

I can't figure out how to make a widget that I can put in getHeaderWidgets() be dynamic and update based on filters set in the resource table. Polling rate is fine as well. The stats depends on the filter value in the resource table. Ie. a week number, which then shows a certain amount of table rows, then the stats in the widget should only calculate for the shown rows in the table.

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/resources/widgets#accessing-page-table-data-in-the-widget
?
